include/bluetooth/hci.h

```diff
@@ -101,6 +101,25 @@ struct bt_hci_cmd_hdr {
 #define BT_LE_FEAT_BIT_CHAN_SEL_ALGO_2          14
 #define BT_LE_FEAT_BIT_PWR_CLASS_1              15
 #define BT_LE_FEAT_BIT_MIN_USED_CHAN_PROC       16
+#define BT_LE_FEAT_BIT_CONN_CTE_REQ             17
```

Comment:
Unrelated to this PR, but I wonder why we decided to have "BIT" in the define name. The non-LE defines don't have that either, and it'd help make these a bit shorter.